.bazelrc

```diff
@@ -53,6 +53,7 @@ build:ci --repository_cache=/home/runner/repo-cache/
 build:ci --flaky_test_attempts=2
 build:ci --color=yes
 build:ci --disk_cache=
+build:ci --@io_bazel_rules_go//go/config:race
```

Comment:
mind setting it for `build:workflows` too while you're here?

Reply:
Done.